client/foundries.go

```diff
@@ -682,10 +682,53 @@ func (a *Api) FactoryPatchConfig(factory string, cfg ConfigCreateRequest) error
 func (a *Api) FactoryListConfig(factory string) (*DeviceConfigList, error) {
 	url := a.serverUrl + "/ota/factories/" + factory + "/config/"
 	logrus.Debugf("FactoryListConfig with url: %s", url)
-	return a.DeviceListConfigCont(url)
+	return a.FactoryListConfigCont(url)
 }
 
 func (a *Api) FactoryListConfigCont(url string) (*DeviceConfigList, error) {
+	// A short cut as it behaves just the same
+	return a.DeviceListConfigCont(url)
+}
+
+func (a *Api) GroupCreateConfig(factory, group string, cfg ConfigCreateRequest) error {
+	data, err := json.Marshal(cfg)
+	if err != nil {
+		return err
+	}
+
+	url := a.serverUrl + "/ota/factories/" + factory + "/device-groups/" + group + "/config/"
```

Comment:
So, a config filename is specified in the POST body, not in URL?

Reply:
Yes

Reply:
and there could be multiple filenames at once